app/narwos.py

```diff
@@ -273,6 +277,52 @@ def show_clusters():
     
     return 'Nothing to show here'
 
+@app.route('/show_filters_submit', methods=['POST'])
+def show_filter_submit():
+
+    show_filter = request.form.get('show_filter')
+    if show_filter:
+        return redirect(url_for('show_filter', show_filter=show_filter))
+    else:
+        return redirect(url_for("index", message=f"Cannot show the filtering!"))
+
+@app.route('/show_filters', methods=['GET'])
+def show_filter():
+
+    df = pd.read_csv(PATH_TO_CURRENT_DF)
```

Comment:
Finally this file will be .gzip.pqrquet in order to provide quick reading and saving

Reply:
And do not use function from pandas, check our utils and search for read_file()

Reply:
Read file do not work for this file extension idk why. Please check the function for returning data for clustering vizualization- Logic looks the same